jujugui/static/gui/src/app/jujulib/index.js

```diff
@@ -57,9 +57,17 @@ var module = module;
       }
       callback(error, data);
     };
-    var failure = function(xhr) {
-      var data = JSON.parse(xhr.target.responseText);
-      var error = data.Message || data.message || data.Error || data.error;
+    var failure = function(xhrOrMessage) {
```

Comment:
I moved the jujulib into the tree last week so that it now gets properly transpiled :)